intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -185,9 +190,12 @@ def log_message(self, msg):
         # Read msg-body and add as raw to a new report.
         # now it's up to a parser to do the interpretation of the message.
         if raw_msg:
-            report = self.new_report()
-            report.add("raw", raw_msg)
-            self.send_message(report)
+            try:
+                report = self.new_report()
+                report.add("raw", raw_msg)
+                self.send_message(report)
+            except:
```

Comment:
This does not trigger the error.

However, if there's any Exception inside this function, sleekxmpp will be kicked from the server because it would send something weird (probably the traceback?). For (better) error messages and debugging, I've put the whole function into an try-except block to log these exceptions.